### PR TITLE
add 'how to reach' and postponed notice

### PR DESCRIPTION
**Pull request by Ankurk99**
Fixes:

### CODE REVIEW

**Comment by Ankk98** [comment]
Replace How to reach to venue page.
Also you have mentioned aqua in place of blue line. Fix it.
@Ankurk99

**Comment by Ankk98** [comment]
@Ankurk99 Check if changing colour of to be announced make it better or not.

**Comment by Ankk98** [comment]
Fix merge conflicts.

**Comment by Ankk98** [comment]
@Ankurk99 Any progress?

**Comment by Ankurk99** [comment]
Done

**Comment by Ankurk99** [comment]
> Replace How to reach to venue page.
> Also you have mentioned aqua in place of blue line. Fix it.
> @Ankurk99
It's on aqua line(comes under noida metro NMRC).

**Comment by Ankurk99** [comment]
> @Ankurk99 Check if changing colour of to be announced make it better or not.

Done

**Comment by Ankurk99** [comment]
@Ankk98 Done all the changes. PTAL

**Comment by Ankk98** [comment]
@Ankurk99 Needs some fixes, ptal at image below.
trident-bug

**Comment by Ankurk99** [comment]
> @Ankurk99 Needs some fixes, ptal at image below.
> trident-bug

Yes, the commit crashed some of the content. On it.

**Comment by Ankurk99** [comment]
Done. PTAL

**Comment by Ankk98** [comment]
1. [ ] Fix footer
2. [ ] Replace this line with - 
"New Delhi/Ghaziabad/Faridabad Nearest Metro Station(Magenta Line or Aqua-Line - Noida Metro)"
 -> "New Delhi/Ghaziabad/Faridabad/Gurugram Nearest Metro Station(Magenta Line or Blue line)"

 @Ankurk99  Check preview Here: [preview](https://deploy-preview-49--tridentjodc.netlify.com/venue.html)

screenshot

**Comment by Ankurk99** [comment]
@Ankk98 PTAL

**Comment by Ankk98** [comment]
Looks good to me. @rexagod review and merge.

File: css/style.css [hunk 1]
```diff
@@ -17,6 +18,9 @@ body {
 div.popupDiv {
   word-wrap: break-word;
 }
+=======
+  font-family: "Roboto Mono", -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji"; }
+>>>>>>> parent of 5882d54... add 'How to reach' to 'Venue' page
```
Comment (rexagod) [inline css/style.css]: A merge conflict was `add`ed and `commit`ed here. Please fix this.

**Review by rexagod — CHANGES_REQUESTED** [review]

**Comment by Ankurk99** [comment]
@rexagod  PTAL.

**Review by rexagod — APPROVED** [review]

**Pull request merged**